packages/paste-core/components/disclosure/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
```

Comment:
This file gets automatically generated. I don't imagine committing this would break anything, but it might be safer to remove it and let it be built automatically?

Reply:
I kind of wanted to try and see if this works partly so we don't have to do a fast follow on adding the changelog back in after it's been released. As an example the Tabs page doesn't have one, we forgot to fast follow.

If this works, we'll never have that problem going forward

Reply:
Oh interesting, kk